src/v2/wallet.js

```diff
@@ -1135,6 +1136,28 @@ Wallet.prototype.prebuildAndSignTransaction = function(params, callback) {
   }).call(this).asCallback(callback);
 };
 
+Wallet.prototype.bumpTransaction = function(params, callback) {
+  return co(function *() {
+    // TODO(BG-9349): change the last check to > 0 and the error message once platform allows multiple transactions to
+    //                be bumped in the same CPFP transaction
+    if (_.isUndefined(params.cpfpTxIds) || !Array.isArray(params.cpfpTxIds) || params.cpfpTxIds.length !== 1) {
+      const error = new Error('expecting cpfpTxIds to be an array of length 1');
+      error.code = 'cpfptxids_not_array';
+      throw error;
+    }
+
+    if (_.isUndefined(params.cpfpFeeRate) && _.isUndefined(params.maxFeeRate)) {
+      debug('neither cpfpFeeRate nor maxFeeRate set in bumpTransaction');
+    }
+
+    if (_.isUndefined(params.maxFee)) {
+      debug('maxFee not set in bumpTransaction');
```

Comment:
@tylerlevine not setting `maxFee` here is potentially dangerous. Not setting `cpfpFeeRate` or `maxFeeRate` above is also potentially dangerous. A user trying to bump a transaction using CPFP might not set these and then end up with a transaction that is a lot higher than they expect. They still have to make another call to send the transaction though, so maybe this is a non-issue. If we wanted to make this more noisy, without requiring these fields, how would we? Do you think this should be more noisy in the first place?

Reply:
Don't we enforce a default max fee rate on platform? I had to introduce an `ignoreMaxFeeRate` flag on platform to get around this for the v1 implementation.

Reply:
```
if (_.isUndefined(maxFeeRateInput)) {
  maxFeeRate = Infinity;
}
```

It would seem not.

Reply:
```js
if (feeRate > process.config.client.constants.maxFeeRate && !ignoreMaxFeeRate) {
    throw new api.Invalid('fee rate too high: ' + (feeRate * 1000) + ' satoshis / KB');
}
```

(transactions.js:1902)

Reply:
That's for v1.

Reply:
You're right, my bad. I'm surprised we aren't enforcing a max fee on v2.